.env

```diff
@@ -0,0 +1,11 @@
+NETWORK=testnet
+XRPL_SEED_COLD="sEdSJKWREQBpGQK71c4QFgD7dUdH9ra"
+XRPL_PUBLICKEY_COLD="ED5E40F69B269C217A3B8BA886DF11ABAD15498D09D9D98A0F0E4CD83BDAD282A0"
+XRPL_PRIVATEKEY_COLD="ED331B43D68F0180212727785B733057D6A3C3A14ADEDE64475A442E6DB3ED253D"
+XRPL_ADDRESS_COLD="rG4LjchUEAMU6trvCVfacqrPvkra7ogtss"
+XRPL_SEED_HOT="sEdTKQhqBH6uko8PMSWS5aL9Av4rCh6"
+XRPL_PUBLICKEY_HOT="EDAF9D781C57A0DAE53F1D93ABB6A90C13F4CD30571C6A3C1E9BE4F8B44080DC79"
+XRPL_PRIVATEKEY_HOT="ED0DACA834F52047B06E9FE10F86D2F0151191226EF3694BD7B73CB7B4EA35CE00"
+XRPL_ADDRESS_HOT="rnXfoueFK2iZ2ub3d42ZKppk5oRWesbc7A"
+MONGODB_BC_URI=mongodb+srv://admin:OXPtL03t56qU8lNO@brainripple.xuavqyp.mongodb.net/?retryWrites=true&w=majority
```

Comment:
.env files should not go into the repository as they are visible to the public. for example we would not want anyone messing with our database.
you can add a .env.example with comments on what the values should be, add .env to .gitginore, delete the .env and then share the actual .env in our private discord channel

Reply:
I created an env.example file and removed the .env file from the repository.